zerver/tests/test_realm.py

```diff
@@ -1305,17 +1305,13 @@ def test_invalid_email_notifications_batching_period_setting(self) -> None:
 
     def test_ignored_parameters_in_realm_default_endpoint(self) -> None:
         params = {"starred_message_counts": orjson.dumps(False).decode(), "emoji_set": "twitter"}
-        json_result = self.client_patch("/json/realm/user_settings_defaults", params)
-        self.assert_json_success(json_result)
+        result = self.client_patch("/json/realm/user_settings_defaults", params)
+        self.assert_json_success(result, ["emoji_set"])
```

Comment:
This API would feel more readable if we made the second argument a mandatory kwarg, i.e. `result, ignore_parameters_unsupported=["emoji_set"])`.

(I think that's a good general software engineering principle to keep in mind -- if you can make it possible to read a function call without having to look at the definition of a function, it's often worth doing; and this is a case where it's not clear from this line what that second list is)